cmd/minio/main.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"log/slog"
+	"os"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/s3"
+	"github.com/dhis2-sre/im-manager/pkg/instance"
+	"github.com/minio/minio-go/v7"
+	"github.com/minio/minio-go/v7/pkg/credentials"
+)
+
+func main() {
+	ctx := context.Background()
+
+	minioEndpoint := "localhost:9000"
+	minioAccessKey := "dhisdhis"
+	minioSecretKey := "dhisdhis"
+	minioBucket := "dhis2"
+	minioUseSSL := false
+
+	s3Region := "eu-west-1"
+	s3Bucket := "im-databases-feature"
```

Comment:
The S3 bucket name should be based on the environment.

Reply:
This was my initial playground for this functionality. It should probably just be deleted

Reply:
Reminder to delete this, if not needed.